lib/active_encode/filename_sanitizer.rb

```diff
@@ -8,6 +8,8 @@ def sanitize_input(input_url)
 
     def sanitize_base(input_url)
       filepath = input_url.is_a?(URI::HTTP) ? input_url.path : input_url
+      # Remove query string if input includes one
+      filepath = filepath.gsub(/\?.*/, '')
```

Comment:
I think this should only apply to http uris since it is possible for file uris to have question marks (e.g. `"file:///home/app/avalon/tmp/file_with_a?_question_mark.mp3"`).  I would have thought that `input_url.path` would have stripped the query string for http urls, but I'm guessing that `input_url` is a string and not an actual `URI::HTTP`.  Maybe attempting to parse `input_url` first thing would work?
```
  filepath = URI.parse(input_url) rescue input_uri
```
I think tests with a variety of inputs could help prove out an implementation.